src/routes/api/prof/+server.ts

```diff
@@ -1,22 +1,17 @@
 import { json } from "@sveltejs/kit";
 import Listener from "../../../models/listener.schema";
 
-interface Listener {
-  pbKey: string;
-  rid: string;
-}
-
 export async function PATCH({ request }) {
   const body = await request.json();
-  const { pbKey, rid } = body;
+  const { pbKey } = body;
   const profanityEnabledStatus = body.profanityEnabled;
   console.log('PATCH /api/prof/:pbKey called with body:', body);
   try {
     let room;
 
     if (profanityEnabledStatus === undefined) {
       // If profanityEnabled is not in the body, find the room and return its state
-      room = await Listener.findOne({ rid: rid }, { profanityEnabled: 1, _id: 0 });
+      room = await Listener.findOne({ pbKey: pbKey }, { profanityEnabled: 1, _id: 0 });
```

Comment:
ooh yeah we did this cus @doniverse was having issues with public key based queries

Reply:
Sure sure u can change it back

Reply:
noice , btw love what you did. im looking thru the code making some svelte reactivity changes. first load was giving me undefined. 

also updated the prod database with the title field 
![image](https://github.com/RobiMez/sma/assets/53259730/62eac782-1802-479c-94e8-28ab47cd1b20)